volmaster/daemon.go

```diff
@@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
 	r := mux.NewRouter()
 
 	postRouter := map[string]func(http.ResponseWriter, *http.Request){
-		"/request":      d.handleRequest,
-		"/create":       d.handleCreate,
-		"/copy":         d.handleCopy,
-		"/mount":        d.handleMount,
-		"/mount-report": d.handleMountReport,
-		"/unmount":      d.handleUnmount,
-		"/remove":       d.handleRemove,
+		"/global":                           d.handleGlobalUpload,
+		"/volumes/create":                   d.handleCreate,
+		"/volumes/copy":                     d.handleCopy,
+		"/volumes/remove":                   d.handleRemove,
+		"/volumes/removeforce":              d.handleRemoveForce,
+		"/volumes/request":                  d.handleRequest,
+		"/mount":                            d.handleMount,
+		"/mount-report":                     d.handleMountReport,
+		"/policies/{policy}":                d.handlePolicyUpload,
+		"/policies/delete/{policy}":         d.handlePolicyDelete,
+		"/runtime/{policy}/{volume}":        d.handleRuntimeUpload,
```

Comment:
similar to mount/unmount may be `/runtime/{policy}/{volume}` should be `/volumes/runtime/{policy}/{volume}` ?